dvc/command/data_sync.py

```diff
@@ -145,6 +147,12 @@ def add_parser(subparsers, _parent_parser):
         default=False,
         help="Pull cache for subdirectories of the specified directory.",
     )
+    pull_parser.add_argument(
+        "--verify",
+        action="store_true",
+        default=False,
+        help="Verify downloaded files checksums.",
+    )
```

Comment:
For the record: better to only provide per-remote config option for now instead of CLI.